src/components/legend/draw.js

```diff
@@ -402,12 +402,12 @@ function repositionLegend(gd, traces) {
         else {
             // approximation to height offset to center the font
             // to avoid getBoundingClientRect
-            textY = tHeight * (0.3 + (1-tLines) / 2);
-            text.attr('y',textY);
-            tspans.attr('y',textY);
+            textY = tHeight * (0.3 + (1 - tLines) / 2);
+            text.attr('y', textY);
+            tspans.attr('y', textY);
```

Comment:
if you're using `eslint` you can use `$ eslint --fix` to fix *all* of the whitespace issues across the codebase automatically